### PR TITLE
handleOrProxy opts arg is not required

### PR DESCRIPTION
**Pull request by jwolski**
@Raynos

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -832,12 +832,12 @@ RingPop.prototype.handleOrProxy =
                 key: key,
                 url: req && req.url
             });
-            this.proxyReq(_.extend(opts, {
+            this.proxyReq(_.extend({
```
Comment (Raynos) [inline index.js]: this is not the semantics you want.

You wanted `_.extend(opts || {}, { ... })`

If you set `{ key: 'foo' }` then other stuff will break horribly downstream.

**Comment by Raynos** [comment]
Recommended: minor

File: index.js (hunk 1 above)
Comment (jwolski) [inline index.js]: true

**Comment by Raynos** [comment]
legit. lgtm.

**Pull request merged**